src/matUtils/main.cpp

```diff
@@ -295,6 +300,10 @@ void extract_main (po::parsed_options parsed) {
         }
         MAT::save_mutation_annotated_tree(subtree, output_mat_filename);
         fprintf(stderr, "Completed in %ld msec \n\n", timer.Stop());
+        wrote_output = true;
+    }
+    if (!wrote_output) {
+        fprintf(stderr, "WARNING: No output files requested!\n");

```

Comment:
Are you sure you only want to give a warning and not throw an error?

Reply:
Want to think about how I'm handling this a bit more, I can concieve of a future where you want to run something without creating an explicit output file maybe, but maybe that's stretching it. and if I'm going to go ahead and make it an error I should probably put it at the top as a long conditional. will change in the next day or two